Vizzuality/mangrove-atlas
# Add restoration sites

### Overview

Added restoration sites including widget, layer list (used to be bottom right


### Designs

https://github.com/globalmangrovewatch/gmw-users/issues/241

### Testing instructions

- click Categories in bottom menu
- click Restoration & Conservation in modal
- click Mangrove Restoration Sites widget to make it exposed
  - there should be a count of restoration sites in the widget
- click the info icon beside the layer toggle (the contents of the info modal will be done later. @lhilarides  knows about it)
- toggle the layer on
- click the map button in the bottom bar
  - note the markers in the map (Nigeria and Samoa currently)
  - The Samoan markers should cluster when zoomed out
- click an unclustered marker to see a map popup for restoration sites (zoom in to see a unclustered marker if necessary 

Note: 
- this layer is unaffected by 'Places' selection.
- `REACT_APP_MRTT_BASE_URL=https://mrtt-api-test-3.herokuapp.com/api/v2/dashboard` will need to be added to `.env` to add the public url to fetch restoration sites.


I cannot comment on the regression potential of this change and will leave that to the repo custodian. I would speculate its quite significant given the centralized configurations required to add a layer.

### Feature relevant tickets

_Link to the related task manager tickets_

---

## Checklist before submitting

      deploying to staging/production, please add brief testing instructions
      to the deploy checklist
